### PR TITLE
[feat]: Updates on the landing page

### PR DESCRIPTION
**Pull request by MarianaKomen**

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -5,7 +5,8 @@
 /* styles/globals.css */
 
 body {
-  background: #222222;
```
Comment (mog-rn) [inline app/globals.css:8]: Do we really need this style here?

File: app/layout.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { NavBar } from '@/app/components'
+import { NavBar } from '@/components'
 import './globals.css'
 import { useState } from 'react'
 import { Inter } from 'next/font/google'
-import Footer from '@/app/components/Elements/Footer'
+import Footer from '@/components/Elements/Footer'
 
 const inter = Inter({ subsets: ['latin'] })
```
Comment (mog-rn) [inline app/layout.tsx:7]: You can remove this font.

File: app/layout.tsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { NavBar } from '@/app/components'
+import { NavBar } from '@/components'
 import './globals.css'
 import { useState } from 'react'
```
Comment (mog-rn) [inline app/layout.tsx:3]: We are not using useState here so you can also remove this.

File: app/layout.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 import { Inter } from 'next/font/google'
```
Comment (mog-rn) [inline app/layout.tsx:4]: Remove this import too.

File: app/page.tsx [hunk 5]
```diff
@@ -1,17 +1,24 @@
...
   return (
     <div>
       <Hero />
       <AboutUs />
-      <Services />
+      <Partnerships/>
       <Cards />
+      <Careers/>
+      <Blog />
+      {/* <Services />
```
Comment (mog-rn) [inline app/page.tsx:20]: Remove these commented stuff.

File: components/Elements/NavBar/index.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
+"use client"
+import Image from 'next/image'
+import Link from 'next/link'
+import { ReactComponentElement, useState} from 'react'
+import { AiOutlineArrowRight, AiOutlineMenu, AiOutlineClose } from 'react-icons/ai'
+
+
+const navLinks = [
```
Comment (mog-rn) [inline components/Elements/NavBar/index.tsx:8]: In the NavBar folder create a links.ts file and move this json there. You can then eport it likeso:

`export const navLinks = [
    // ... rest of the json objects here
]`

File: components/Elements/NavBar/index.tsx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    url:'contact'
+  },
+]
+export const NavBar = () => {
+  const [nav, setNav] = useState(false)
+  const handleClick = () => setNav(!nav)
+  return (
+    <div className='text-white text-lg h-auto min-w-full drop-shadow-lg absolute z-10'>
+      <div className='flex items-center justify-between '>
+        <div className=''>
```
Comment (mog-rn) [inline components/Elements/NavBar/index.tsx:46]: Remove this empty className property

File: components/Elements/NavBar/index.tsx [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+         <Image
+         src='/assets/images/FLAG.png'
+         alt='Kenyan flag'
+         width={65}
+         height={182}
+         />
+        </div>
+
+        <div className=''>
```
Comment (mog-rn) [inline components/Elements/NavBar/index.tsx:55]: Remove empty className property

File: components/Elements/NavBar/index.tsx [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+          {/* map through the array to display each link */}
+          <div className='md:px-2 md:text-sm lg:px-4 lg:text-lg'>
+            {navLinks.map(navLink=>(
+              <Link key={navLink.id} href={navLink.url} className="mx-2 lg:mx-4 text-[14px]">{navLink.title}</Link>
+            ))}
+          </div>
+        </div>
+        
+        <div className='hidden md:flex justify-between items-center md:space-x-2 lg:space-x-6 mr-8'>
+          {/* <div>
```
Comment (mog-rn) [inline components/Elements/NavBar/index.tsx:74]: Remove this commented code.

File: components/Elements/NavBar/index.tsx [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+        </div>
+      </div>
+      
+      <div className={!nav ? 'hidden' : 'flex flex-col items-center justify-center bg-black text-lg border-b-2 border-green-100'}>
+            {navLinks.map(link=>(
+                <Link key={link.id} href={link.url} className='py-4 xl:hidden '>{link.title}
+                </Link>
+            ))}
+            <div className='flex flex-col mb-4 md:hidden'>
+                    {/* <button className='border-none my-4 xl:px-8 py-[8px] outline-none text-white rounded-md'>
```
Comment (mog-rn) [inline components/Elements/NavBar/index.tsx:101]: Remove this commented code. We don't need login on this landing page

File: components/Landing/Cards/index.tsx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+        {/* CARDS */}
+        
+        <div className='mx-auto px-6 md:pt-8 md:grid grid-cols-3 gap-8'>
+            <div className='w-[350px] mx-auto md:w-auto bg-white shadow-xl flex flex-col p-4 my-8 rounded-lg hover:scale-105 duration-300'>
+                    <Image className='w-20 mx-auto mt-[-2rem] md:mt-2'
+                    src='/assets/icons/card1-icon.png' alt='card-1 icon' width={84} height={84}
+                    />
+                    <h2 className='text-2xl text-green-600 font-bold text-center py-4 uppercase'>Safety</h2>
+                    <p className='text-center text-black p-2 text-[14px]'>Our riders are vetted. Safety begins with you and everyone in  the society is tasked with their safety and those around them.</p>
+                    {/* <Link href='/' className='text-red-500 text-center py-4 flex flex-row justify-center items-center gap-[0.5rem]'>
```
Comment (mog-rn) [inline components/Landing/Cards/index.tsx:18]: Remove this commented code in this file.

File: components/Landing/Partnerships/index.tsx [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+                </div>
+            </div>
+
+            {/* row-3 */}
+            <div className='flex ml-24 space-x-4'>
+            <div className="-mt-6 w-[153px] h-[77.30px] bg-stone-50 rounded-[30px] shadow-2xl flex justify-center items-center">
+                    <div className="w-[107.39px] h-[107.39px] relative ">
+                        <Image
+                        src='/assets/images/SONGA-small-logo.png'
+                        alt=''
```
Comment (mog-rn) [inline components/Landing/Partnerships/index.tsx:82]: Add the alt text to the images.

**Review by mog-rn — CHANGES_REQUESTED** [review]
Please have a look at the changes I have suggested and work on  them.

File: app/globals.css (hunk 1 above)
Comment (MarianaKomen) [reply app/globals.css:8]: No, we dont

Comment (MarianaKomen) [reply app/globals.css:8]: Done

File: app/layout.tsx (hunk 2 above)
Comment (MarianaKomen) [reply app/layout.tsx:7]: Done

File: app/layout.tsx (hunk 3 above)
Comment (MarianaKomen) [reply app/layout.tsx:3]: Done

File: app/layout.tsx (hunk 4 above)
Comment (MarianaKomen) [reply app/layout.tsx:4]: Done

File: app/page.tsx (hunk 5 above)
Comment (MarianaKomen) [reply app/page.tsx:20]: Done

File: components/Elements/NavBar/index.tsx (hunk 8 above)
Comment (MarianaKomen) [reply components/Elements/NavBar/index.tsx:55]: Done

File: components/Elements/NavBar/index.tsx (hunk 9 above)
Comment (MarianaKomen) [reply components/Elements/NavBar/index.tsx:74]: Done

File: components/Elements/NavBar/index.tsx (hunk 10 above)
Comment (MarianaKomen) [reply components/Elements/NavBar/index.tsx:101]: Done

File: components/Landing/Cards/index.tsx (hunk 11 above)
Comment (MarianaKomen) [reply components/Landing/Cards/index.tsx:18]: Done

File: components/Landing/Partnerships/index.tsx (hunk 12 above)
Comment (MarianaKomen) [reply components/Landing/Partnerships/index.tsx:82]: Done

**Review by mog-rn — COMMENTED** [review]
There are a few things you haven't finished working on.

**Pull request merged**